notebooks/finetuning.ipynb

```diff
@@ -0,0 +1,137 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Objective Finetuning Quickstart\n",
+    "\n",
+    "Improve your Indexes by integrating your own data with Finetuning. Objective Finetuning amplifies the capabilities of AI search by incorporating your unique insights — your data and your feedback.\n",
+    "\n",
+    "Read the full [Quickstart Doc](https://www.objective.inc/docs/quickstarts/finetuning-quickstart)\n",
+    "\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "Begin by obtaining an API key and installing the SDK:\n",
+    "\n",
+    "To get an API key, [join the waitlist](https://objective.inc/contact).\n",
+    "\n",
+    "```bash\n",
+    "pip install objective-sdk\n",
```

Comment:
Should we put this in a code block so they can run it in the notebook? e.g. 
`! pip install objective-sdk`